util/Sequencer/Sequencer.gd

```diff
@@ -28,13 +31,9 @@ func sequence(sequence: SongSequence):
 				})
 			var dur = note.note_start + note.duration
 			if dur > song.length: song.length = dur
-	player = AnimationPlayer.new()
-	add_child(player)
+
 	player.add_animation("song", song)
 	player.root_node = self.get_path()
```

Comment:
```suggestion
```
this line is not needed as the player is the child of this note. root node is set automatically

Reply:
Ah yeah good point